### PR TITLE
[linux] Import kmfl

### PR DESCRIPTION
**Pull request by glasseyes**
import from CVS on sourceforge to personal repo then updates from debian packages that are more recent 

add build.sh, README.md and history.md
now import to keyman

### CODE REVIEW

File: linux/history.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Keyman for Linux Version History
+
+## 2018-04-23
```
Comment (darcywong00) [inline linux/history.md]: I think this initial stage is just

```
## 11.0 alpha
```

(skip the date until this goes to 11.x beta).

Also, maybe include what version of KMFL you're importing.

File: linux/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Keyman for Linux
```
[the remark this answers is not in the document]
Comment (glasseyes) [reply linux/README.md]: Done, and added status for TC builds

**Comment by darcywong00** [comment]
Some of the `linux/build/agent/` files were copied from WeSay, but still talk about building WeSay.

**Comment by darcywong00** [comment]
@mcdurdin  Can we move `windows/src/LICENSE.md` to the root folder (update date to -2018) and have it apply to all the products?

**Comment by mcdurdin** [comment]
@darcywong00 not at this time. There's a bunch of things to consider, e.g. KMFL is GPL, others are MIT, and we have other included code around the place that is also different (although compatible). It's not a trivial task :)

File: README.md [hunk 3]
```diff
@@ -39,4 +39,4 @@ Each platform maintains multiple types of releases:
 
 Copyright (c) 2018 SIL International. All rights reserved.
 
-Licensed under the [MIT](./windows/src/LICENSE.md) License.
\ No newline at end of file
+Licensed under the [MIT](./windows/src/LICENSE.md) License.
```
Comment (mcdurdin) [inline README.md]: This needs to stay but we need to list the exception for Linux at present I think.

File: README.md [hunk 4]
```diff
@@ -19,6 +20,7 @@ Open Source
 ## Desktop Platforms ##
 [Windows](./windows/src/README.md)
 [macOS](./mac/README.md)
+[L](./linux/README.md)
```
[the remark this answers is not in the document]
Comment (glasseyes) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (glasseyes) [reply README.md]: added "[KMFL](./linux) is licensed under GNU General Public License as published by the Free Software Foundation; either [version 2](./linux/libkmfl/COPYING) of the License, or (at your option) any later version."

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (glasseyes) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (glasseyes) [reply README.md]: That was done for this PR then #870 does the change of licensing

**Pull request merged**